### PR TITLE
BTFS-1697

### PR DESCRIPTION
**Pull request by steveyeom**

### CODE REVIEW

**Comment by steveyeom** [comment]
Updated PR to use Reed-Solomon option

File: shell_test.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"crypto/md5"
 	"fmt"
+	files "github.com/ipfs/go-ipfs-files"
```
Comment (taiyangc) [inline shell_test.go]: why not "go-btfs-files"

**Review by taiyangc — COMMENTED** [review]
one concern, the rest is good.

File: shell_test.go (hunk 1 above)
Comment (steveyeom) [reply shell_test.go]: i tried but it reports errors like no path.. it may be related to go-btfs-api's lower go-ipfs-files v0.0.1.

Comment (taiyangc) [reply shell_test.go]: Yeah, it should not have any lower go-ipfs-files dependencies.. otherwise we have not truly fixed the folder issue.

Comment (steveyeom) [reply shell_test.go]: Updated the imports.

**Review by taiyangc — APPROVED** [review]
lgtm

**Pull request merged**